database/migrations/20191218223245_users.js

```diff
@@ -1,8 +1,24 @@
-
 exports.up = function(knex) {
+    return knex.schema.createTable('users', users => {
+      users.increments();
   
-};
-
-exports.down = function(knex) {
+      users
+        .string('username', 255)
+        .notNullable()
+        .unique();
   
-};
+      users.string('password', 255).notNullable();
+  
+      users
+        .string('email', 255)
+        .notNullable()
+        .unique();
+  
+      users.string('date_created', 255).notNullable();
+    });
+  };
+  
+  exports.down = function(knex) {
+    return knex.schema.dropTableIfExists('users');
+  };
+  
```

Comment:
Think about adding a name field in the users table that way the User page can be personalized on the FE

Reply:
Ahh didn't even think of that. Will add it in.